python/llm/src/ipex_llm/transformers/low_bit_linear.py

```diff
@@ -213,8 +214,12 @@ def ggml_convert_qtype(tensor: torch.Tensor, qtype: int,
                       f"Last dim of input tensor must be multiple of {QK}")
 
     dst_size = (n // QK) * block_size_in_bytes
-    dst_tensor = torch.empty(dst_size, dtype=torch.uint8,
-                             device=device)
+    if enable_deepspeed_zero3:
+        dst_tensor = torch.empty(dst_size, dtype=torch.bfloat16,
```

Comment:
If using `torch.bfloat16`, I think we only need half the `dst_size`, right? (assuming `dst_size` is a even number)